### ISSUE TITLE
openblt via CAN on Proteus-F7

### ISSUE THREAD

**Issue by andreika-git** [rusefi/rusefi#6218]
TLDR; The communication via CAN between BootCommander.exe and OpenBLT on Proteus seems to be ok, but for some reason the utility is unhappy and doesn't start uploading the firmware:
  ```
Connecting to target bootloader...[TIMEOUT]
Attempting backdoor entry (reset system if this takes too long)...
```
More info:
- the bootloader itself works on F7 (one can normally update the firmware via USB);
- CAN frames are transmitted in both directions (there are signals on both Rx and Tx of the transceiver);
- packets are successfully received in the bootloader in the CanReceivePacket() function, BLT_TRUE is returned (tested by adding LED blinker to the function);
- packets are successfully sent back from the bootloader in the CanTransmitPacket() function (tested by adding LED blinker to the function);
- MSG_ID for Tx and Rx are handled correctly. The bitrate is also correct. If I change the IDs or bitrate on the PC side, the communication stops;
- the same behavior is with external 12v power without USB connection;
- the command line arguments are the same as on the working small-can-board or 2chan.
